addons/grass/functions/script_component.hpp

```diff
@@ -0,0 +1 @@
+#include "\z\afm\addons\grass_force\script_component.hpp"
```

Comment:
```suggestion
#include "\z\afm\addons\grass\script_component.hpp"
```
```suggestion
#include "..\script_component.hpp"
```
(recently I'm leaning towards relative include in function as it makes refactoring easier.)

Reply:
It should be changed in all addons then? Maybe separately?

Reply:
Use this in new components, adapt in old ones if anything is changed.